contracts/GnosisSafe.sol

```diff
@@ -11,18 +11,19 @@ contract GnosisSafe {
     string public constant NAME = "Gnosis Safe";
     string public constant VERSION = "0.0.1";
 
+    // masterCopy always needs to be first declared variable, to ensure that it is at the same location as in the Proxy contract.
+    // It should also always be ensured that the address is stored alone (uses a full word)
     GnosisSafe masterCopy;
-    uint8 public threshold;
+
     uint256 public nonce;
+    uint8 public threshold;
```

Comment:
The declaration of the threshold was moved to ensure that it is not combined with the masterCopy address